### ISSUE TITLE
honeycombing

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6693]
We should use the Fleisher definition for this term?
https://hpo.jax.org/app/browse/term/HP:0025175

Honeycombing represents destroyed and fibrotic lung tissue containing numerous cystic airspaces with thick fibrous walls, representing the late stage of various lung diseases, with complete loss of acinar architecture. The cysts range in size from a few millimeters to several centimeters in diameter, have variable wall thickness, and are lined by metaplastic bronchiolar epithelium. On chest radiographs, honeycombing appears as closely approximated ring shadows, typically 3-10 mm in diameter with walls 1-3 mm in thickness, that resemble a honeycomb; the finding implies end-stage lung disease. On CT scans, the appearance is of clustered cystic air spaces, typically of comparable diameters on the order of 3-10 mm but occasionally as large as 2.5 cm. Honeycombing is usually subpleural and is characterized by well-defined walls (54). It is a CT feature of established  pulmonary fibrosis. Because honeycombing is often considered specific for pulmonary fibrosis and is an important criterion in the diagnosis of usual interstitial pneumonia, the term should be used with care, as it may directly impact patient care.

PMID:18195376

Comment
See Figure 25 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Comment by pnrobinson** [comment]
Might not be synonym. @mgriese please help define difference between honeycombing and honeycomb lung

**Comment by mgriese** [comment]
Honeycombing is the process leading to a honeycomb lung; a third term in this contex is honeycomb cysts. All terms are equivalent.

**Comment by pnrobinson** [comment]
done
